option java_package = "pl.project13.distmetrics.common.proto";

package distmetrics;

import "common.proto";

message SubscribeRequest {
    required string     resourceId = 1;
    required MetricType metricType = 2;
}

message SubscriptionResponse {
    required int64  subscriptionId = 1;
    required string host           = 2;
    required int32  port           = 3;
}